odbc/src/api/bind_col_tests.rs

```diff
@@ -0,0 +1,333 @@
+mod unit {
+    use crate::{
```

Comment:
related to earlier comment, maybe we should test rebinding a column works. E.g. call bind by column twice with the same number and check that the buffer ptr updates properly?